test_runner/fixtures/pageserver/http.py

```diff
@@ -416,6 +425,11 @@ def timeline_delete(self, tenant_id: TenantId, timeline_id: TimelineId, **kwargs
     def timeline_gc(
         self, tenant_id: TenantId, timeline_id: TimelineId, gc_horizon: Optional[int]
     ) -> dict[str, Any]:
+        """
+        Unlike most handlers, this will wait for the layers to be actually
+        complete their shutdown up until registering them to the deletion
+        queue.
```

Comment:
This is the first time I've heard of layers shutting down.

Please stick to established terminology, this just adds confusion.

Reply:
d2aea5624 will talk just about deletion queue not shutdown.